examples/pylint/w0133_pointless_exception_statement.py

```diff
@@ -0,0 +1,4 @@
+"""Pointless Exception Statement Example"""
+# The exception statement below is not raised, assigned,
```

Comment:
Just say "The exception below", the word 'statement' is a bit misleading since it's really the *expression* that isn't being used